src/include/ndpi_typedefs.h

```diff
@@ -528,6 +528,20 @@ struct ndpi_vxlanhdr {
 } PACK_OFF;
 
 /* ************************************************************ */
+
+#define NDPI_IN_PKT_DIR_UNKNOWN		0
+#define NDPI_IN_PKT_DIR_C_TO_S		1	/* Client -> Server */
+#define NDPI_IN_PKT_DIR_S_TO_C		2	/* Server -> Client */
+
+#define NDPI_FLOW_BEGINNING_UNKNOWN	0
+#define NDPI_FLOW_BEGINNING_SEEN	1
+#define NDPI_FLOW_BEGINNING_NOT_SEEN	2
```

Comment:
Can please give some more information what `UNKNOWN`, `SEEN` and `NOT_SEEN` exactly means?
Any kind of doxygen compatible input appreciated.

Reply:
Ok, I'll add a description.

The basic idea behind these values is to allow the application to NOT pass SYNs packets to the library, without losing any classification capabilities (see #1211)
If the application wants (or is able) it can inform the library that the 3 way handshake packets have not been passed to nDPI but they have been received, i.e. the flows has been received from its beginning

I don't like the names very much... if you have hany proposal...

Reply:
> I don't like the names very much... if you have hany proposal...

Together with the documentation it make sense, I guess. Thanks.